android/authentication.md

```diff
@@ -1,615 +1,704 @@
+   setup`.
 
-   In Android Studio, choose `File > New > Activity > Basic Activity` and type an activity name, such as `AuthenticatorActivity`. If you want to make this your starting activity, move the intent filter block containing `.LAUNCHER` to the `AuthenticatorActivity` in your app's `AndroidManifest.xml`.
+4. Type a display name for your app, select a category for your app from the `Category`
+   drop-down list, and then choose `Create App ID`.
 
-```xml
-<activity android:name=".AuthenticatorActivity">
-    <intent-filter>
-        <action android:name="android.intent.action.MAIN" />
-        <category android:name="android.intent.category.LAUNCHER" />
-    </intent-filter>
-</activity>
-```
+![Image](./images/new-facebook-app-new-app-id.png)
 
-4. Update the `onCreate` function of your `AuthenticatorActivity` to call `AWSMobileClient`. This component provides the functionality to resume a signed-in authentication session. It makes a network call to retrieve the AWS credentials that allows users to access your AWS resources and registers a callback for when that transaction is complete.
 
-If the user is already signed in, the app switches to the `NextActivity`.  If the user isn't signed in, the user is presented with the AWS Mobile configurable sign-in UI.  After the user is authenticated, the app continues to the `NextActivity`.
+5. Complete the `Security Check` that appears. Your new app then appears in the
+   `Dashboard`.
 
-```kotlin
-import android.app.Activity;
-import android.os.Bundle;
+![Image](./images/new-facebook-app-id.png)
 
-import com.amazonaws.mobile.auth.ui.SignInUI;
-import com.amazonaws.mobile.client.AWSMobileClient;
-import com.amazonaws.mobile.client.AWSStartupHandler;
-import com.amazonaws.mobile.client.AWSStartupResult;
+6. Copy the App ID and note it for later when using the Amplify CLI.
 
-class AuthenticatorActivity : Activity() {
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
+![Image](./images/facebook-app-id-console-entry.png)
 
+7. In the Facebook Developer portal's left hand navigation list, choose `Settings`, then
+   choose `+ Add Platform`.
 
-  AWSMobileClient.getInstance().initialize(this) {
-      val signInUI = AWSMobileClient.getInstance().getClient(
-              this@AuthenticatorActivity,
-              SignInUI::class.java) as SignInUI?
-      signInUI?.login(
-              this@AuthenticatorActivity,
-              MainActivity::class.java)?.execute()
-  }.execute()
+![Image](./images/new-facebook-add-platform.png)
 
-}
-```
+8. Choose your platform and provide information about your app that Facebook will use for
+   integration during credential validation.
 
-Choose the run icon (|play|) in Android Studio to build your app and run it on your device/emulator. You should see the ready made sign-in UI for your app. Check out the next steps to learn how to [customize your UI](add-aws-mobile-user-sign-in-customize).
+   `For iOS:`
```

Comment:
This is still an iOS section

Reply:
Removed but as noted in the PR I need to scrub the Facebook and Google sections as they're not reviewed.